### PR TITLE
Add CW23 blog

### PR DESCRIPTION
**Pull request by pherterich**
Adding pic for blog

### CODE REVIEW

**Comment by pherterich** [comment]
@amangoel185 - if you want to add something, latest version is now here.

**Comment by pherterich** [comment]
@NPDebs - future date was not the issue, so whenever you want to go digging for what I did wrong...

**Comment by NPDebs** [comment]
Hi, @pherterich. 👋🏼
The 🪲 has been found.

**Comment by NPDebs** [comment]
After serving up the website locally, the blogpost was present...but the hero image was missing.

The picture being used has a local path (it is a file found in the website, rather than somewhere else on the web)...so, the `image` variable is used to display it.
The downside is that you no longer have an alt text. :(

Unless you have some other changes to make, this looks ready for a merge.

**Comment by pherterich** [comment]
Super late in catching up on this. I can't merge this yet, can anyone approve please or let me know what needs fixing (if there are changes required to fit in with all the new fancy updates) please? Not sure if @NPDebs has approval rights, so maybe something @bebatut needs to do...

**Comment by NPDebs** [comment]
> Super late in catching up on this. I can't merge this yet, can anyone approve please or let me know what needs fixing (if there are changes required to fit in with all the new fancy updates) please? Not sure if @NPDebs has approval rights, so maybe something @bebatut needs to do...

Better late than never. 😉

Tagging @bebatut. 🙏🏻

File: _posts/2023-06-28-OLS-at-CollabW23.md
Comment (bebatut) [inline _posts/2023-06-28-OLS-at-CollabW23.md:38]: It might be good to have an introduction sentence to mention that now different people will share their experience. It is not really clear for me from the title

File: _posts/2023-06-28-OLS-at-CollabW23.md
Comment (bebatut) [inline _posts/2023-06-28-OLS-at-CollabW23.md:53]:
```suggestion
This year's Collaboration Workshop has been undoubtedly the highlight of my year so far. 
```

File: _posts/2023-06-28-OLS-at-CollabW23.md
Comment (bebatut) [inline _posts/2023-06-28-OLS-at-CollabW23.md:57]:
```suggestion
As Gemma mentioned, I shared a workshop session with her, where I showcased my OLS project, titled ["The Undergraduate's Guide To Research Software Engineering"](https://software.ac.uk/cw23/mini-workshops-and-demo-sessions#3.4b), which received a warm reception from the attendees. 
```

File: _posts/2023-06-28-OLS-at-CollabW23.md
Comment (bebatut) [inline _posts/2023-06-28-OLS-at-CollabW23.md:58]:
```suggestion
All in all, the workshop was both productive and rejuvenating, and I'm looking forward to similar events in the future.
```

**Review by bebatut — COMMENTED** [review]
Thanks @pherterich. Just put some small comments. Feel free to ignore them and I can merge

File: _posts/2023-06-28-OLS-at-CollabW23.md
Comment (pherterich) [reply _posts/2023-06-28-OLS-at-CollabW23.md:38]: Good point - added something now!

**Review by bebatut — APPROVED** [review]
Looks good. Thanks @pherterich and other for the post

**Pull request merged**
